packages/libs/error-stack/tests/test_hooks.rs

```diff
@@ -11,23 +11,22 @@ use common::*;
 use error_stack::Report;
 
 #[test]
+#[allow(deprecated)]
 fn debug() {
     Report::set_debug_hook(|_report, fmt| fmt.write_str("debug hook")).expect("Unable to set hook");
     assert_eq!(format!("{:?}", create_report()), "debug hook");
 
-    let err = Report::set_debug_hook(|_, _| Ok(())).expect_err("Could set the hook twice");
-    assert_eq!(format!("{:?}", err.current_context()), "HookAlreadySet");
+    Report::set_debug_hook(|_, _| Ok(())).expect("Able to overwrite hook");
```

Comment:
This cannot fail anymore.

```suggestion
    Report::set_debug_hook(|_, _| Ok(())).expect("Could not set the hook twice");
```